### ISSUE TITLE
Failure to send 8 GB directory

### ISSUE THREAD

**Issue by nemobis** [warner/magic-wormhole#63]
```
$ wormhole --version
magic-wormhole 0.8.0
$ wormhole send 104OLYMP
Building zipfile..
main function encountered error
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 1274, in unwindGenerator
    return _inlineCallbacks(None, gen, Deferred())
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 1128, in _inlineCallbacks
    result = g.send(result)
  File "/usr/lib/python2.7/site-packages/wormhole/cli/cmd_send.py", line 51, in go
    d = self._go(w)
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 1274, in unwindGenerator
    return _inlineCallbacks(None, gen, Deferred())
--- <exception caught here> ---
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 1128, in _inlineCallbacks
    result = g.send(result)
  File "/usr/lib/python2.7/site-packages/wormhole/cli/cmd_send.py", line 63, in _go
    offer, self._fd_to_send = self._build_offer()
  File "/usr/lib/python2.7/site-packages/wormhole/cli/cmd_send.py", line 215, in _build_offer
    num_files += 1
  File "/usr/lib64/python2.7/zipfile.py", line 801, in __exit__
    self.close()
  File "/usr/lib64/python2.7/zipfile.py", line 1347, in close
    " would require ZIP64 extensions")
zipfile.LargeZipFile: Central directory offset would require ZIP64 extensions
```

**Comment by warner** [comment]
Huh. Does anyone know how to make zipfile.py add those extensions?

**Comment by warner** [comment]
This was fixed by the patch in #138 . The won't-fit-in-space problem still remains (#58).
